etabotsite/etabotapp/views.py

```diff
@@ -186,8 +186,17 @@ def get(self, request, format=None):
             status=status.HTTP_200_OK)
 
 
+def jira_callback(request):
+    """Receieve authorization code from JIRA OAuth."""
+    """Provided as a query parameter called code. This code can be
+    exchanged for an access token"""
+    logging.info(request.GET)
+    # logging.info(request.GET.get('code'))
+    return render(request, 'jira_callback.html')
```

Comment:
I don't see any code that connect with JIRA to get access token. In general, it's good practice to finish a feature before a PR is submitted.

Reply:
this is first phase of development where I use info from request.GET with the code that I exchange manually with curl.
This is a big feature so I am breaking it down into small ones.

Reply:
I'll add TODO comment section so it's clear what's next.